x-pack/plugins/observability_solution/observability/public/pages/slo_details/components/overview/overview.tsx

```diff
@@ -129,27 +123,50 @@ export function Overview({ slo }: Props) {
           title={i18n.translate('xpack.observability.slo.sloDetails.overview.tagsTitle', {
             defaultMessage: 'Tags',
           })}
-          subtitle={
-            slo.tags.length > 0 ? (
-              <EuiFlexGroup
-                direction="row"
-                alignItems="flexStart"
-                gutterSize="s"
-                responsive={false}
-                wrap
-              >
-                {slo.tags.map((tag) => (
-                  <EuiFlexItem grow={false} key={tag}>
-                    <EuiBadge color="hollow">{tag}</EuiBadge>
-                  </EuiFlexItem>
-                ))}
-              </EuiFlexGroup>
-            ) : (
-              <EuiText size="s">-</EuiText>
-            )
-          }
+          subtitle={<TagsList tags={slo.tags} />}
         />
         {IndicatorOverview}
+        {'index' in slo.indicator.params && (
+          <OverviewItem
+            title={i18n.translate('xpack.observability.slo.sloDetails.overview.indexTitle', {
+              defaultMessage: 'Index pattern',
+            })}
+            subtitle={slo.indicator.params.index}
+          />
+        )}
+        {'filter' in slo.indicator.params && (
+          <OverviewItem
+            title={i18n.translate('xpack.observability.slo.sloDetails.overview.overallQueryTitle', {
+              defaultMessage: 'Overall query',
+            })}
+            subtitle={
+              <DisplayQuery
+                query={slo.indicator.params.filter}
+                index={slo.indicator.params.index}
+              />
+            }
+          />
+        )}
+        {'good' in slo.indicator.params && (
+          <OverviewItem
+            title={i18n.translate('xpack.observability.slo.sloDetails.overview.goodQueryTitle', {
+              defaultMessage: 'Good query',
+            })}
+            subtitle={
+              <DisplayQuery query={slo.indicator.params.good} index={slo.indicator.params.index} />
```

Comment:
Do you think there's any benefit to including the good queries for opinionated SLIs like synthetics and apm? Perhaps a switch statement where you can have constant values representing the queries used for apm and synthetics, that defaults back to `slo.indicator.params.good` for the custom indicators.

Reply:
possibly i guess, but we can do in a follow up